Library/Homebrew/formula.rb

```diff
@@ -31,6 +31,8 @@
 require "extend/on_system"
 require "api"
 require "extend/api_hashable"
+require "cli/parser"
+require "commands"
```

Comment:
I don't think we need these?

Reply:
@Bo98 @hyuraku can one of you submit a PR to remove if so?